firefox: update to 139.0

MingcongBai opened the pull request:
Topic Description
-----------------

- firefox: update to 139.0
    Track patches at AOSC-Tracking/firefox @ aosc/v139.0
    \(HEAD: 78fbb48ee608f230655ac69bd8245f898081e5ce\).

Package(s) Affected
-------------------

- firefox: 139.0

Security Update?
----------------

Yes, #11194 

Build Order
-----------

```
#buildit firefox
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

MingcongBai committed:
firefox: update to 139.0

Track patches at AOSC-Tracking/firefox @ aosc/v139.0
(HEAD: 78fbb48ee608f230655ac69bd8245f898081e5ce).

Fearyncess commented:
![image](https://github.com/user-attachments/assets/3ca8b0bd-4542-4903-9fe4-2ae0645eb725)
LGTM on AMD64

MingcongBai commented:
Does not launch on LoongArch...

xen0n commented:
LoongArch startup crash issue: https://github.com/loongson-community/discussions/issues/93

chenx97 committed:
firefox: backport some regression fix patches

These patches fix segfaults on loongarch64, loongson3 and riscv64.

Track patches at AOSC-Tracking/firefox @ aosc/v139.0
(HEAD: 2304293d342f947f6860545b52cb70cbe84bbafb).

MingcongBai committed:
firefox: disable "Awesome Bar" hand-off

Such that search input in the new tab page from the user would not jump to
the address bar (which felt weird).

MingcongBai committed:
topics: add firefox-139.0.toml

chenx97 commented:
111111

chenx97 reviewed (approved):
we revive RISC-style ISA from segfaults 11111
